src/lib/stores/titleQueue.ts

```diff
@@ -0,0 +1,32 @@
+import { writable } from "svelte/store";
+
+export interface TitleUpdate {
+	convId: string;
+	title: string;
+}
+
+const queueStore = () => {
```

Comment:
Why do you need a queue?

Reply:
In case we end up with multiple updates for multiple conversations at the same time somehow, it's better handled as a queue

Reply:
This seems like an edge case no? I would have kept it as simple as possible.

Reply:
see https://github.com/huggingface/chat-ui/pull/512/commits/59d978a27bd6dd34b49370bb42487737eee1c031